server/internal/usecase/interfaces/scene.go

```diff
@@ -57,3 +60,16 @@ type UpdateClusterParam struct {
 	Name       *string
 	PropertyID *id.PropertyID
 }
+
+type AddStyleInput struct {
+	SceneID id.SceneID
+	Name    *string
+	Value   *scene.StyleValue
+}
```

Comment:
```suggestion
type AddStyleInput struct {
	SceneID id.SceneID
	Name    string
	Value   *scene.StyleValue
}
```
if the name is required